services/app-web/src/components/DownloadButton/DownloadButton.tsx

```diff
@@ -1,25 +1,28 @@
 import React from 'react'
-import { Link } from '@trussworks/react-uswds'
+import { ActionButton } from '../ActionButton'
 
 type DownloadButtonProps = {
     text: string
-    zippedFilesURL: string
+    zippedFilesURL: string | undefined
 }
 
 export const DownloadButton = ({
     text,
     zippedFilesURL,
 }: DownloadButtonProps): React.ReactElement => {
+    const handleClick = () => {
+        if (zippedFilesURL) {
+            window.open(zippedFilesURL)
```

Comment:
why are we opening the link with JS? Shouldn't it be a simple Link in the end? Id worry about screen readers here going outside the normal semantics

Reply:
Oh! Good call.

Reply:
@macrael I updated this to use the `Link` component again. From this [blog](https://css-tricks.com/how-to-disable-links/), disabling the link might not be the way to go, so I'm rendering just a div with the loading spinner until the link is ready. Could use you're eyes on this again.